arch/Kconfig

```diff
@@ -281,6 +281,28 @@ config KOBJECT_TEXT_AREA
 	help
 	  Size of kernel object text area. Used in linker script.
 
+config KOBJECT_DATA_AREA_RESERVE_MULTIPLIER
+	int "Multiplier for reserving kobject data area"
+	default 200
```

Comment:
shouldn't this be 2 instead? or you really want to say 200 times larger?

Reply:
200%. So it is possible to specify 120% or 150%. Kconfig doesn't take numbers with decimals.

Reply:
Then apparently it should say "percent" somewhere in the description. And maybe even in the option name, and then maybe use standard percentage's additive semantics, where [+]0% means "zero extra overhead" and [+]100% means "2x times the original size".

Reply:
Updated.